db/migrate/20180215112954_fix_whitehall_remove_change_note.rb

```diff
@@ -0,0 +1,15 @@
+class FixWhitehallRemoveChangeNote < ActiveRecord::Migration[5.1]
+  def up
+    d = Document.where(content_id: "b3b96cb9-e7fa-493c-87a5-31d71d79c7a7").first
+    change_notes = d.editions.map(&:change_note).compact
+    change_notes.select{ |change_note| change_note.note == "Added funding allocations for individual LAs for 2018 to 2019."}
+    # Re-present editions to content store
+    puts "The editions that need to be represented downstream are: #{change_notes.map(&:edition_id)}"
```

Comment:
It is actually actually possible to represent stuff from the migration, see https://github.com/alphagov/publishing-api/blob/fab6a131d52b9759160b4004f9e11953dad50ba7/db/migrate/20171011154900_fix_remaining_specialist_publisher_first_published_at_dates.rb#L13 for some prior art here. An important thing to note is the `disable_ddl_transaction!` at the top.

Reply:
Thanks!